### PR TITLE
Fix/scottish database fields

### PR DESCRIPTION
**Pull request by freesteph**
This ended up being much more than just the database fields because the corresponding ActiveRecord mappings needs to be updated at the same time. The diff is huge because of a couple of `sed` passes on the application code so I would recommend looking at the commits separately.

The app boots.

### CODE REVIEW

**Comment by pixeltrix** [comment]
Can you convert the .env file as well:

```
COMPOSE_PROJECT_NAME=scottish-petitions
```

https://github.com/ScottishParliament/e-petitions/blob/main/.env

**Comment by freesteph** [comment]
> Can you convert the .env file as well:
> 
> ```
> COMPOSE_PROJECT_NAME=scottish-petitions
> ```
> 
> https://github.com/ScottishParliament/e-petitions/blob/main/.env

Thanks! Was actually wondering where that identifier lived :smile:

**Review by cdccollins — APPROVED** [review]

File: .github/workflows/preview.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ jobs:
 
     - name: Deploy application
       env:
-        APPSIGNAL_APP_NAME: welsh-petitions-preview
```
Comment (pixeltrix) [inline .github/workflows/preview.yml:104]: This should be `scottish` not `gaelic`

File: lib/package_builder.rb [hunk 2]
```diff
@@ -245,7 +245,7 @@ def package_path
   end
 
   def profile
-    ENV.fetch('AWS_PROFILE', 'welsh-petitions')
+    ENV.fetch('AWS_PROFILE', 'gaelic-petitions')
```
Comment (pixeltrix) [inline lib/package_builder.rb:248]: This should be `scottish`

File: lib/package_builder.rb [hunk 3]
```diff
@@ -283,7 +283,7 @@ def notify_appsignal
   end
 
   def appsignal_app_name
-    ENV.fetch('APPSIGNAL_APP_NAME', "welsh-petitions-#{environment}")
+    ENV.fetch('APPSIGNAL_APP_NAME', "gaelic-petitions-#{environment}")
```
Comment (pixeltrix) [inline lib/package_builder.rb:286]: This should be `scottish`

File: lib/package_builder.rb [hunk 4]
```diff
@@ -299,7 +299,7 @@ def region
   end
 
   def release_bucket
-    "welsh-petitions-deployments"
+    "gaelic-petitions-deployments"
```
Comment (pixeltrix) [inline lib/package_builder.rb:302]: This should be `scottish`

**Review by pixeltrix — CHANGES_REQUESTED** [review]

**Pull request merged**